inst/extdata/temp-data-inject/L201.nonghg_steepness.csv

```diff
@@ -1,8 +1,8 @@
-INPUT_TABLE
```

Comment:
I will copy the file from the old system into this folder.  I may have opened this file in Excel (accidently!) which does crazy stuff to the file.